### PR TITLE
SWSS-34: Implement html emails

### PR DESCRIPTION
**Pull request by ihordubas**
#34 
- Sign up email
- Reset password email
- Email for confirmation of email changing
image
image
image

### CODE REVIEW

File: server/app/templates/emails/change_email.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+                font-size: 18px; 
+                text-decoration: none;
+            }
+        </style> 
+    </head>
+    <body>
+        <div class="email">
+            <h1 class="email-header">SheetWithoutShit</h1><hr>
+            <h1>Hi, {username}.</h1>
+            <p>We would like to confirm that you prefer using {new_email} as your email.</p>
```
Comment (PetrushynskyiOleksii) [inline server/app/templates/emails/change_email.html:38]: Can we emphasize for new email value? For example, add a bold style.

File: server/app/templates/emails/change_email.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+            <p>In case you don't want to change your current email - ignore this message.</p>
+            <p>In order to confirm the email changing, you need to click the button below.<p/>
+            <p>Note that this link is valid for 48 hours.</p>
+            <p>After the time limit has expired, you will have to resubmit the request for an email changing.</p>
+            <button id="change-email-button">
+                <a href={change_email_url}>Confirm</a>
+            </button>
+        </div>
+    </body>
+</html>
```
Comment (PetrushynskyiOleksii) [inline server/app/templates/emails/change_email.html:48]: Please add a new line for all created files.

**Review by PetrushynskyiOleksii — CHANGES_REQUESTED** [review]

File: server/app/templates/emails/change_email.html (hunk 1 above)
Comment (ihordubas) [reply server/app/templates/emails/change_email.html]: Done

File: server/app/templates/emails/change_email.html (hunk 2 above)
Comment (ihordubas) [reply server/app/templates/emails/change_email.html]: Done

**Review by PetrushynskyiOleksii — APPROVED** [review]

**Pull request merged**